src/components/Header/Header.css

```diff
@@ -0,0 +1,126 @@
+.header {
+  width: 90%;
+  height: 4rem;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background-color: #131a17;
+  padding: 0 10%;
```

Comment:
This padding value for header is causing the page to be extra-wide so we are going to have a horizontal scroll.
![image](https://github.com/dianagafita/Portfoliu/assets/46847960/1955a3a3-4316-4042-bf73-02e0f7de17d2)


Please have a look at CSS Box model.

https://www.youtube.com/watch?v=rIO5326FgPE